utils/acpica/Makefile

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=acpica
+PKG_VERSION:=20170531
+PKG_RELEASE:=1
+
+PKG_BASENAME:=$(PKG_NAME)-unix
+PKG_SOURCE:=$(PKG_BASENAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://acpica.org/sites/$(PKG_NAME)/files/$(PKG_SOURCE)
```

Comment:
Normally `PKG_SOURCE_URL` contains only dirname part of the complete url, excluding `$(PKG_SOURCE)`